### DISCUSSION TITLE
Italian loacalization

### DISCUSSION THREAD

**Discussion by bovirus** [Sophia-Community/SophiApp#169] [Q&A]
### Your Idea

Italian language update

### Windows Version

Windows 10 2022H2

### Comments

Hi.

Can I update italian language?

What relation is theer between your sofwtare and Windows 10/Windows 11 Debloater?

Thanks.

**Comment by TheBossMagnus** [comment]
Comments

> Can I update italian language?

Hi, I'm the translator for the Italian language of SophiApp. Yes! feel free to pr any changes([here all the instruction](https://github.com/Sophia-Community/SophiApp#translating)). There are some fixes to do especially in the ToolTips

> What relation is theer between your sofwtare and Windows 10/Windows 11 Debloater?
Do you mean SophiaScript?

**Comment by bovirus** [comment]
Hi.

I posted in my github reposiotry fork the changes about xaml file.

https://github.com/bovirus/SophiApp/blob/master/src/SophiApp/Localizations/IT.xaml

I foudn som typo errors and some italian strings localized as english workflow.
The enhglish workflow is differnt then italian workflow (adjective/verbs/agents have a differnt positions).

I wil, post soon also the JSON file.

**Reply by farag2** [reply]
Take only this JSON: https://github.com/Sophia-Community/SophiApp/blob/master/src/SophiApp/Localizations/UIData_IT.json
What about to create a PR?

**Comment by farag2** [comment]
@TheBossMagnus, what do you think?

**Reply by TheBossMagnus** [reply]
I agree with some changes, but others are equivalent to my translation. I'll wait for a pr

**Comment by TheBossMagnus** [comment]
Could you please make a pull request so we can propose/discuss changes in a more efficient way

**Comment by bovirus** [comment]
@TheBossMagnus 

Made the PR for .xaml file.

For other files (json) next step.
